crates/metrics/src/importer.rs

```diff
@@ -15,14 +18,19 @@ pub struct ImporterMetrics {
     pub block_height: Gauge,
     pub latest_block_import_timestamp: Gauge<f64, AtomicU64>,
     pub execute_and_commit_duration: Histogram,
+    pub gas_per_block: Histogram,
+    pub fee_per_block: Histogram,
+    pub transactions_per_block: Histogram,
 }
 
 impl Default for ImporterMetrics {
     fn default() -> Self {
         let block_height_gauge = Gauge::default();
         let latest_block_import_ms = Gauge::default();
-        let execute_and_commit_duration =
-            Histogram::new(timing_buckets().iter().cloned());
+        let execute_and_commit_duration = Histogram::new(buckets(Buckets::Timing));
+        let gas_per_block = Histogram::new(buckets(Buckets::GasUsed));
+        let fee_per_block = Histogram::new(buckets(Buckets::Fee));
+        let transactions_per_block = Histogram::new(buckets(Buckets::TransactionsCount));
 
         let mut registry = global_registry().registry.lock();
```

Comment:
should we have sub-registries, one for each module?

Reply:
No, the idea is that all metrics are using on global registry isntead of creating global registry per module. It simplifies the dumping of all metrics

Reply:
Regardless if we should or not I wouldn't push it in this PR, if you don't mind. I'll create a follow-up issue so we can discuss on grooming.

Reply:
Ok, I'm closing the issue. I created it literally a few seconds before @xgreenx posted his answer :)